python/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
+
+set(Python_ADDITIONAL_VERSIONS ${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})
```

Comment:
I guess this has been moved here to let the user control which version `find_package(PythonInterp)` uses. I suggest to put this under the "# find Python..." comment, with a comment what this line does.